test/com/arpnetworking/metrics/portal/TestUtil.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2016 Groupon.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics.portal;
+
+import com.google.common.base.Optional;
+import models.internal.Alert;
+import models.internal.Expression;
+import models.internal.NagiosExtension;
+import org.joda.time.Period;
+
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Utility class for tests.
+ *
+ * @author Deepika Misra (deepika at groupon dot com)
+ */
+public final class TestUtil {
```

Comment:
No. This is far to generic. Please split the functionality up into more appropriate classes.